### DISCUSSION TITLE
Minimize Docker Size

### DISCUSSION THREAD

**Discussion by shresthaujjwal** [raxod502/riju#119] [Q&A] [answered]
Hi @raxod502 is it possible to minimize the Docker size. As per your setup documentation it says 40gb. I just need couple language support will that help reduce the Docker size ?

**Comment by raxod502** [comment, marked as answer]
Yes, you can simply build only the languages that you want. There's a Docker image per language, and the total size is around 40 GB for all of them. If you only need the app server plus a few languages, it would only be a few GB.

In the absence of https://github.com/raxod502/riju/issues/109, you can just delete the YAML files for the languages you don't need, and follow the standard instructions. See also https://github.com/raxod502/riju/issues/122.
